en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -1018,412 +1018,6 @@ Versioned::set_reading_mode($origMode);
 
 See [Reading versions by stage](#reading-versions-by-stage) for more about using reading modes.
 
-## Using the history viewer
```

Comment:
Need to document the new way to do it, rather than just deleting the docs altogether. This is still a valid thing for people to want to do.

Reply:
Turns out you can just add `HistoryViewerField` to your form and it works.

Reply:
This still hasn't been updated? It should be pretty quick docs to write. Just a short php example adding that field to a new `Root.History` tab.